src/main/java/flab/delideli/service/CartServiceImpl.java

```diff
@@ -12,26 +13,34 @@
 @AllArgsConstructor
 public class CartServiceImpl implements CartService {
 
-    private final CartDao cartDao;
-
-    @Override
-    public void insertCart(AddCartDTO addCartDTO, String userId) {
-        cartDao.insertCart(addCartDTO,userId);
-    }
-
-    @Override
-    public List<CartlistDTO> getCartList(String userId) {
-        List<CartlistDTO> cartlist= cartDao.getCartList(userId);
-        return cartlist;
-    }
-
-    @Override
-    public void deleteCart(String userId, int cartId) {
-        cartDao.deleteCart(userId, cartId);
-    }
-
-    @Override
-    public void deleteCartItem(String userId, int cartItemId) {
-        cartDao.deleteCartItem(cartItemId);
-    }
+	private final CartDao cartDao;
+
+	@Override
+	public void insertCart(AddCartDTO addCartDTO, String userId) {
+		cartDao.insertCart(addCartDTO, userId);
+	}
+
+	@Override
+	public List<CartlistDTO> getCartList(String userId) {
+		List<CartlistDTO> cartlist = cartDao.getCartList(userId);
+		return cartlist;
+	}
+
+	@Override
+	public void deleteCart(String userId, int cartId) {
+		cartDao.deleteCart(userId, cartId);
+	}
+
+	@Override
+	public void deleteCartItem(String userId, int cartItemId) {
+		cartDao.deleteCartItem(cartItemId);
+	}
+
+	@Override
+	public boolean confirmUser(String userId, int cartItemId) {
+		if (cartDao.getCartOwnerId(cartItemId).equals(userId)) {
+			return true;
+		}
+		return false;
```

Comment:
이 부분은 아래처럼 간소화가 가능할 것 같네요~

```suggestion
		return cartDao.getCartOwnerId(cartItemId).equals(userId);
```

Reply:
추가적으로 `cartDao.getCartOwnerId(cartItemId)`가 null일 가능성이 있을까? 고민해보면 좋을 것 같아요.
NullPointerException의 가능성이 있으니까요~

Reply:
네, 이 부분도 생각해보겠습니다~